src/domexpression.js

```diff
@@ -31,6 +37,21 @@ var _ = Mavo.DOMExpression = $.Class({
 			this.expression = expression;
 		}
 
+		if (this.attribute?.startsWith("mv-attr-")) {
+			this.originalAttribute = this.attribute;
+			this.attribute = this.attribute.replace("mv-attr-", "");
+
+			if ([SVG_NAMESPACE_URI, MATHML_NAMESPACE_URI].includes(this.element.namespaceURI)) {
+				this.attribute = Mavo.getProperAttributeCase(this.element, this.attribute);
+			}
+
+			this.fallback ??= Mavo.Primitive.getValue(this.element, {attribute: this.attribute});
+			let expression = this.element.getAttribute(this.originalAttribute);
+			this.element.removeAttribute(this.originalAttribute);
```

Comment:
There seems to be a lot of repetition between this and the preceding block implementing `mv-value`. I'll push a commit to address it.

Reply:
Whoa! Cool! It's working with this beautiful piece of code. 🥳 Thank you!!